install/kubernetes/cilium/charts/hubble-tls/templates/_helpers.tpl

```diff
@@ -0,0 +1,29 @@
+{{/*
+Generate TLS certificates for Hubble server and Hubble Relay.
+*/}}
+{{- define "ca.gen-certs" }}
+{{- $ca := .ca | default (genCA "hubble-ca.cilium.io" 1095) -}}
```

Comment:
I don't understand this line. `.ca` is never set anywhere, is it? Can we just use 

```
{{- $ca := genCA "hubble-ca.cilium.io" 1095 -}}
```

here?

Reply:
It's a trick to generate the CA certs only once. Without this, every template invocation would result in a different CA and thus every certs that is generated would be signed by a different CA which would be quite problematic.

Reply:
Ahh, that makes sense. Thanks for the clarification! If you are rebasing, could you add a short comment in Helm source mentioning this (so nobody accidentally "cleans" up the assignment)

Reply:
Done.